library/thrift

```diff
@@ -4,4 +4,6 @@
 0.9.3: git://github.com/ahawkins/docker-thrift@d322572f7dd6ea468a14a4d832fbec26f152c71e 0.9
```

Comment:
Any particular reason to keep this version listed here?  Removing it here will remove it from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding it on the official build server), but the tag will still be available to users who want it.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

https://thrift.apache.org/download at least seems to imply that only the latest (0.11) series is actually actively supported, but maybe there's a better resource about that elsewhere?